### DISCUSSION TITLE
can I provide multiple compilerPath?

### DISCUSSION THREAD

**Discussion by phatpaul** [microsoft/vscode-cpptools#10157] [Q&A] [answered]
My team and I are split between windows and linux for development machines.  I would like to use a single configuration in c_cpp_properties.json which would work on either host.  But it seems I need to specify the compiler path in order to resolve the system includes (like <stdint.h> ).
Ideally I would just provide a list of paths for intellisense to try:
```
"compilerPath": [
                "C:/nxp/MCUXpressoIDE_11.6.0_8187/ide/tools/bin/arm-none-eabi-gcc",
                "/usr/local/mcuxpressoide-11.6.0_8187/ide/tools/bin/arm-none-eabi-gcc"
            ],
```
The compiler is not in the PATH, and there is no other environment variable on the host with the path.  I would be open to referencing a .env file or something that is not committed to vcs that points to the compiler.

**Comment by sean-mcmanus** [comment, marked as answer]
We don't allow multiple compiler paths. If you have configurationProvider set to something like `ms-vscode.cmake-tools` and you're using CMake then it can be configured based on the compiler you have on my machine. You could open a feature request. Or have 2 configurations one for Windows and one for Linux.
